articles/cms/wordpress/configuration.md

```diff
@@ -35,15 +39,15 @@ First, we'll check for the Client created for your WordPress site.
 
 1. **Client Type** must be set to **Regular Web Application**
```

Comment:
You already say on the first step to select regular web app as type, so we can remove this.

Reply:
Where I say it on step 1 is if someone is creating a new client. If they're troubleshooting an existing one then this needs to match.

Reply:
In this case I'd expect to see that in a Troubleshooting section, under the relevant error

Reply:
To provide a little more context here ... the Clients created when WordPress is installed are created as they need to be when the installer is run. The problem we run across is that Clients created a year ago might need to be updated in a few ways (this attribute being one of them). We try to do this automatically but sometimes that process fails and the updates need to be done manually. If that's the case, we direct folks to this Configuration page and say "make sure your client matches this." The errors that they experience run the gamut, in fact it would be incredibly difficult to know what, exactly, they would see in each case. If someone is reading this, they skip the part about creating a new Client and, if this is missing, they skip that configuration as well. 

TL; DR: the repetition is not harmful, IMHO, and is necessary in a very common support use case.

Reply:
@mpaktiti - Any thoughts on this?

Reply:
I was missing the context, it makes sense now. Sorry for the delay, I'm merging now.